tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -489,7 +489,7 @@ EOF
     # Workaround for bsc#1199448
     my $node_helper = 'node-helper.yaml';
     assert_script_run("curl " . data_url("virt_autotest/kubevirt_tests/$node_helper") . " -o $node_helper");
-    assert_script_run("sed -e \"s/$kubevirt_ver/\${KUBEVIRT_VERSION}/g\"");
+    assert_script_run("sed -i 's/\${KUBEVIRT_VERSION}/$kubevirt_ver/g'");
```

Comment:
No filename specified in `sed` command, does it work?

Reply:
Sorry, my mistake. Updated.